### DISCUSSION TITLE
I can't hack my account! What do I do?

### DISCUSSION THREAD

**Discussion by Socoolboi** [Prodigy-Hacking/ProdigyMathGameHacking#1227] [Questions] [answered]
I can't get my 'Wills Cheat Menu' to work! What should I do?

**Comment by donaldli2020** [comment, marked as answer]
in the custom pnp url do prodigy-hacking.hostedposted.com
type the password nootnoot and save

**Reply by Socoolboi** [reply]
OK, but the html 'prodigy-hacking.hostedposted.com' does not work.

**Reply by Socoolboi** [reply]
Also please send a link to the custom pnp url .

**Reply by donaldli2020** [reply]
https://prodigy-hacking.hostedposted.com/game.min.js

**Reply by S0vereignity** [reply]
that works. I tried it. Just ignore the message at the top.
